deploy/clusterrole.yaml

```diff
@@ -33,6 +33,18 @@ rules:
 - apiGroups: ["ptp.openshift.io"]
   resources: ["*"]
   verbs: ["*"]
+- apiGroups: ["monitoring.coreos.com"]
```

Comment:
Please also update the same for 4.4 manifest - install.spec.clusterPermissions`: https://github.com/openshift/ptp-operator/blob/master/manifests/4.4/ptp-operator.v4.4.0.clusterserviceversion.yaml#L129